tests/rados/test_mon_connection_scores.py

```diff
@@ -153,14 +253,27 @@ def run(ceph_cluster, **kw):
                 "Proceeding to check mon connection scores after restarting mon service"
             )
 
+            # Mon map is not updated for restart scenarios, hence omitting the Mon map update check.
```

Comment:
Looks like a issue? 

Once the mon is restarted, and it comes back up, the mon will request for elections, and post new election, there should be a new epoch? 

We can collect the logs and check with dev.